components/molecule/thumbnail/src/index.scss

```diff
@@ -0,0 +1,101 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$bgc-molecule-thumbnail-basic: $c-white !default;
+
+$sizes-molecule-thumbnail: (
+  large: 144px,
+  medium: 72px,
+  small: 48px,
+  xsmall:  40px
+) !default;
+
+$ratios-molecule-thumbnail: (
+  1: 1,
+  4: 3,
+  16: 9
+) !default;
+
+$bd-outer-molecule-thumbnail: 1px solid color-variation($c-gray, 3) !default;
+$bdc-outer-hover-molecule-thumbnail: $c-text-link !default;
+$bd-inner-molecule-thumbnail: 3px solid $c-white !default;
+$bgc-molecule-thumbnail: $c-gray-light !default; 
+$bs-molecule-thumbnail: 0 !default;
+
+a {
```

Comment:
please avoid using element selector, better use classes. You might need to change something here when using the link factory